t/predictable-errors.t

```diff
@@ -0,0 +1,33 @@
+use strict;
+use warnings;
+
+use JSON::Validator;
+use Math::Permute::List;
+use Test::More;
+
+my $jv = JSON::Validator->new();
+permute {
```

Comment:
I don’t see what value this adds. What’s wrong with a simple for-loop?

Reply:
With a simple for loop I'd have to write out all the permutations by hand, this creates them on the fly.

Reply:
It's sort of irrelevant, since it is "randomized" by `keys()` inside JSON::Validator. I think it would be better to run this block 100 times with a fixed list.

Either way, I don't want the extra dependency just for this.

Reply:
OK, done